bazel/http-server/index.bzl

```diff
@@ -80,6 +83,12 @@ http_server_rule = rule(
         "launcher": "%{name}.sh",
     },
     attrs = {
+        "serving_root": attr.string(
+            doc = """
+            The root path to serve files for the application from, defaults to the current
+            directory of the package.
```

Comment:
OOC why is this needed? Isn't the serving root simply the directory where the rule is defined?

I would strongly prefer to preserve that "mental model" because all other access to "deps", or "assets" depends on that being the case as the "root dir".

Reply:
This should also be already possible via `rootPaths` ?

Reply:
In the case of an application created using architect, a `browser` and `server` directory are each created, and we further in framework put this into a `build` directory.

So we need to perform the serving for that project from `build/browser`

Reply:
I think this can be achieved via `rootDirs` already?

Reply:
Ahh I see it now, I was misunderstanding what it was looking for.